resolver/addressable_resolver.go

```diff
@@ -169,17 +172,17 @@ func (r *URIResolver) URIFromObjectReference(ctx context.Context, ref *corev1.Ob
 		Namespace:  ref.Namespace,
 		Name:       ref.Name,
 	}, parent); err != nil {
-		return nil, apierrs.NewNotFound(gvr.GroupResource(), ref.Name)
+		return nil, multierr.Combine(err, apierrs.NewNotFound(gvr.GroupResource(), ref.Name))
```

Comment:
why not using go 1.13 wrap/unwrap feature?

Reply:
The linter is not happy with this format 
```
fmt.Errorf("%s; %w", err.Error(), apierrs.NewNotFound(gvr.GroupResource(), ref.Name))
```

```
 Error: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors (errorlint)
  Error: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors (errorlint)
  Error: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors (errorlint)
```

Reply:
It should be:

```go
fmt.Errorf("%w: %v", apierrs.NewNotFound(gvr.GroupResource(), ref.Name), err)
```